config/sample_conf.json

```diff
@@ -1,6 +1,6 @@
 {
-    "token": "<SECRET REPLACE ME>",
-    "prefix": "!",
+    "token": "",
```

Comment:
Please read `<SECRET REPLACE ME>` to `token`
Remember, everyone will refer to this sample_conf.json. We need to highlight replacing that.